index2.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
```

Comment:
A note on the name of this HTML file. Typically, `index` is the name we give to the homepage. The other pages can be named after their content, for example `resume` or `projects` or `about-me`

Reply:
In reality, those are tests and not quite sure which version will be the index, so, for now I have 2 versions of the index page.